go.mod

```diff
@@ -1,60 +1,74 @@
 module sigs.k8s.io/apiserver-network-proxy
 
-go 1.17
+go 1.18
 
 require (
-	github.com/golang/mock v1.4.4
-	github.com/golang/protobuf v1.4.3
-	github.com/google/uuid v1.1.2
-	github.com/prometheus/client_golang v1.7.1
-	github.com/spf13/cobra v0.0.3
+	github.com/golang/mock v1.6.0
+	github.com/golang/protobuf v1.5.2
+	github.com/google/uuid v1.3.0
+	github.com/prometheus/client_golang v1.13.0
+	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
-	google.golang.org/grpc v1.42.0
-	k8s.io/api v0.20.10
-	k8s.io/apimachinery v0.20.10
-	k8s.io/client-go v0.20.10
-	k8s.io/component-base v0.20.10
-	k8s.io/klog/v2 v2.4.0
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.0
+	golang.org/x/net v0.0.0-20220805013720-a33c5aa5df48
+	google.golang.org/grpc v1.48.0
+	google.golang.org/protobuf v1.28.1
+	k8s.io/api v0.24.3
+	k8s.io/apimachinery v0.24.3
+	k8s.io/client-go v0.24.3
+	k8s.io/component-base v0.24.3
+	k8s.io/klog/v2 v2.70.1
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.32
```

Comment:
Leave the konnectivity-client at `v0.0.0`, since it's using a replace directive.